bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt

```diff
@@ -39,7 +39,11 @@ import kotlin.jvm.JvmName
  *
  * @param root the root node to be updated
  */
-class ModelImporter(private val root: INode, private val continueOnError: Boolean) {
+class ModelImporter(
```

Comment:
`@param`s for `continueOnError` and `childFilter` are now missing.

Reply:
Added.